docs/release-notes.asciidoc

```diff
@@ -20,6 +20,51 @@ required to enable the UI.
 * <<post-upgrade-req, Enable the process analyzer>>. This is only required if you want to view
 <<alerts-analyze-events, graphical representations of process relationships>>.
 
+[discrete]
+[[release-notes-7.10.0]]
+== 7.10.0
+
+[discrete]
+[[breaking-changes-7.10.0]]
+==== Breaking changes
+
+*Signals template updated for rollover indices*
+
+The `create_index_route` now checks if the template needs to be upgraded
+before creating the index. If the index already exists and the template was upgraded,
+the index rolls over so that the write index has the upgraded mapping.
+This breaks the old mappings that have `risk_score mapped` as a keyword.
+In the new mapping, `signal.rule.risk_score` is a float.  After rolling over,
+there is a conflict between the old and new `signal.rule.risk_score` for some
+features, such as aggregations.
+
+See ({pull}/80019[#80019]) for details.
+
+*Connect incident fields allowed when cases are sent*
+
+You can now specify connector incident fields when cases are sent. This includes:
+* Jira: issue type, priority, and parent issue in the case of a subtask.
+* IBM Resilient: issue types, and severity.
+* ServiceNow: urgency, severity, and impact.
+
+See ({pull}77327[#77327]) for details.
+
+[discrete]
+[[bug-fixes-7.10.0]]
+==== Bug fixes and enhancements
+* Adds Metadata and Discovery Analysis Jobs to Security Integration ({pull}76023[#76023]).
+* Improve Alert Telemetry for the Security app ({pull}77200[#77200]).
+* Allows passwords to be visible on security screens ({pull}77394[#77394]).
+* Groups features for role management ({pull}78152[#78152]).
+* Warns users when security is not configured ({pull}78545[#78545]).
+* Made changes to sharing saved-objects phase 1.5 ({pull}75444[#75444]).
```

Comment:
This should probably be more user-friendly. Suggestion: `Enhancements for saved object management workflows` or something to that effect. Could probably pull `phase 1.5` out.